main.go

```diff
@@ -9,7 +9,7 @@ import (
 	"regexp"
 	"time"
 
-	"github.com/gojp/goreportcard/handlers"
+	"github.com/YotpoLtd/goreportcard/handlers"
```

Comment:
Seems like this got changed by accident, we would need to change it back to `gojp` before merging